docs/source/start.rst

```diff
@@ -257,20 +257,20 @@ gates from the ``pyquil.gates`` module, which allows us to add operations to our
 .. note::
 
     PyQuil also provides a handy function for you to ensure that a local qvm and quilc are currently running in
-    your environment. To make sure both are available you import `from pyquil.api import local_qvm` and then run
-    `local_qvm()`. This will start a qvm and quilc instances using subprocesses if they have not already been started.
+    your environment. To make sure both are available you import `from pyquil.api import local_forest_runtime` and then run
+    `local_forest_runtime()`. This will start qvm and quilc instances using subprocesses if they have not already been started.
```

Comment:
Maybe try to connect to the specified ports first, and if something is running spit out a warning that it's anticipated that these are qvm/quilc processes. (And if that's supported, request something like the version of quilc/qvm to make sure the processes listening understand the protocol and it's actually qvm/quilc?) Otherwise go ahead and start new instances?

Reply:
okay cool, I'll take a look into that

I prototyped sth very quickly, just leave it here incase it's needed at some point in the future

https://gist.github.com/sauercrowd/1a1d7ece4b6dc129700d4126b6aa9e2d